src/find_schema_nodes.js

```diff
@@ -0,0 +1,41 @@
+import { visit } from 'graphql';
+
+/*
+  scopes: (requried) string array
+  schema: (required) GraphQLSchema
+
+  A scope could be a `Type`, `Type.field` or `Type.field.argument`.
```

Comment:
I was really hoping there was a way in `graphql-js` to generate a `path` from a GraphQL schema member ([like this](https://github.com/rmosolgo/graphql-ruby/blob/e881fc30760f6724ddbb627d66f0ee2e2e98def5/lib/graphql/schema/member/has_path.rb)).

This would have allowed us to generate a `path` from a validation error (i.e. `Query.myField`) and then compare that with the ignore list. It would have been less work. 😄 

If there isn't, we'll want to make sure these functions work with non-object types.

For example:

- Enum
- Enum.Value
- Union
- Union.PossibleType
- InputObject
- etc..

For completeness, you could probably [take a look at `graphql-ruby` to see the various cases](https://github.com/rmosolgo/graphql-ruby/blob/e881fc30760f6724ddbb627d66f0ee2e2e98def5/lib/graphql/schema/finder.rb).

Reply:
@cjoudrey Hey 👋 
1. Didn't find a path generator in `graphql-js` unfortunately. AST nodes don't seem to have ancestry information and I see no such API on the schema. The `path` property of an error is for execution only and relates to the path in response.
2. Added support for `Enum.VALUE` as a scope and improved the test to check all supported cases.
Though I skipped `UnionType.PossibleType.aField` case supported by the referred finder as I see no usecase for that.

Reply:
Thanks again @id-ilych! Your changes look good to me. I'll get this merged shortly. 😄

Reply:
@cjoudrey Hello there :wave: How is it going? 😄

Reply:
Oh damn! I completely forgot to get this merged. 🙈 Sorry!

Reply:
This PR is part of v1.0.0. 😄 Thanks again for your contribution!